src/itest/java/uk/gov/companieshouse/insolvency/delta/steps/InsolvencyDataSteps.java

```diff
@@ -44,15 +55,72 @@ public void insolvency_delta_consumer_service_is_running() {
     @When("a {string} with {string} is published to the topic {string} and consumed")
     public void a_with_is_published_to_the_topic_and_consumed(String message, String companyNumber, String topicName)
             throws InterruptedException {
+        startWiremockServer();
         this.companyNumber = companyNumber;
 
-        stubInsolvencyDataApiServiceCalls(companyNumber);
+        stubInsolvencyDataApiServiceCalls(companyNumber, 200);
         kafkaTemplate.send(topicName, createMessage(message));
 
         CountDownLatch countDownLatch = new CountDownLatch(1);
         countDownLatch.await(5, TimeUnit.SECONDS);
     }
 
+    @When("a non-avro message is published and failed to process")
+    public void a_non_avro_message_is_published_and_failed_to_process() throws InterruptedException {
+        startWiremockServer();
+        kafkaTemplate.send(topic, "invalid message");
+
```

Comment:
Cosmetic, probably better to separate error-retry steps and feature files from happy path so that we can highlight different types of integration tests